DI-1801/generate_count_data.py

```
@@ -0,0 +1,136 @@
+
+    overall_count_per_variant = calculate_count_for_all_cancers(
+        genie_data_with_sample_info
+    )
+
+    overall_count_per_variant.to_csv(
+        args.output,
+        index=False,
+    )
+
+    # Overall count per cancer type
+    haemonc_cancer_types = read_txt_file_to_list(args.haemonc_cancer_types)
```

Ruff (0.8.2)

Local variable `haemonc_cancer_types` is assigned to but never used

Remove assignment to unused variable `haemonc_cancer_types`

(F841)